tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -463,6 +457,38 @@ public function deleteAllSpacesOfTheType(string $driveType): void {
 		}
 	}
 
+	/**
+	 * users delete their personal space at the end of the test
+	 *
+	 * @return void
+	 *
+	 * @throws Exception|GuzzleException
+	 */
+	public function deleteAllPersonalSpaces(): void {
+		$query = "\$filter=driveType eq personal";
+		$createdUsers= $this->featureContext->getCreatedUsers();
+
+		foreach($createdUsers as $user)
+		{
+			for ($i = 0; $i < 2; ++$i) {
+
+				$this->theUserListsAllHisAvailableSpacesUsingTheGraphApi(
+					$user["actualUsername"],
+					$query
+				);
+				$drives = $this->getAvailableSpaces();
+
+				foreach ($drives as $value) {
```

Comment:
what if we set the user to be space-admin, then this would also delete the project spaces, or?

Reply:
yes, we should be able to expand this kind of code to also delete project spaces. Let's get this merged first, and then move forward.